src/main/webapp/app/exercises/programming/shared/lifecycle/programming-exercise-lifecycle.component.html

```diff
@@ -133,11 +133,27 @@ <h6 jhiTranslate="artemisApp.assessment.assessment"></h6>
                     [ngStyle]="exercise.assessmentType !== assessmentType.AUTOMATIC || (!isExamMode && !exercise.dueDate) ? { color: 'grey' } : {}"
                     class="form-control-label"
                     for="allowComplaintsForAutomaticAssessment"
-                    jhiTranslate="artemisApp.programmingExercise.timeline.complaintOnAutomaticAssessment"
-                ></label>
+                    >{{ 'artemisApp.programmingExercise.timeline.complaintOnAutomaticAssessment' | artemisTranslate }}</label
```

Comment:
Why did you change that?

Reply:
I misread the guidelines and thought we would choose `artemisTranslate` over `jhiTranslate`, which is actually the other way around

https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html